dotCMS/src/main/java/com/dotcms/publishing/output/BundleOutput.java

```diff
@@ -0,0 +1,69 @@
+package com.dotcms.publishing.output;
+
+import com.dotcms.publishing.PublisherConfig;
+import com.dotcms.util.DotPreconditions;
+import com.dotmarketing.util.Config;
+
+import com.dotmarketing.util.Logger;
+import com.google.common.collect.ImmutableList;
+import com.liferay.util.FileUtil;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+public abstract class BundleOutput implements Closeable {
```

Comment:
comment it

Reply:
done https://github.com/dotCMS/core/pull/20173/commits/e491040a5cc382fa8a25457ba027679b4afa86b7#diff-e65f8fbb893f9aa226ba9b0fff84476edb7ab06338dc40aaa0b92c4a49b306eeR12